src/panels/config/cloud/dialog-cloud-already-connected/dialog-cloud-already-connected.ts

```diff
@@ -0,0 +1,142 @@
+import { formatDateTime } from "../../../../common/datetime/format_date_time";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { createCloseHeading } from "../../../../components/ha-dialog";
+import { haStyleDialog } from "../../../../resources/styles";
+import type { HomeAssistant } from "../../../../types";
+import type { CloudAlreadyConnectedParams as CloudAlreadyConnectedDialogParams } from "./show-dialog-cloud-already-connected";
+import "../../../../components/ha-alert";
+
+@customElement("dialog-cloud-already-connected")
+class DialogCloudAlreadyConnected extends LitElement {
+  public hass!: HomeAssistant;
+
+  @state() private _params?: CloudAlreadyConnectedDialogParams;
+
+  public showDialog(params: CloudAlreadyConnectedDialogParams) {
+    this._params = params;
+  }
+
+  public closeDialog() {
+    this._params?.closeDialog();
+    this._params = undefined;
+    fireEvent(this, "dialog-closed", { dialog: this.localName });
+  }
+
+  protected render() {
+    if (!this._params) {
+      return nothing;
+    }
+    const { details } = this._params;
+
+    return html`
+      <ha-dialog
+        open
+        @closed=${this.closeDialog}
+        .heading=${createCloseHeading(
+          this.hass,
+          this.hass.localize(
+            "ui.panel.config.cloud.dialog_already_connected.heading"
+          )
+        )}
+      >
+        <div>
+          <p>
+            ${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.description"
+            )}
+          </p>
+          <b>
+            ${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.other_home_assistant"
+            )}
+          </b>
+          <div class="instance-details">
+            <div class="instance-detail">
+              <p>
+                ${this.hass.localize(
+                  "ui.panel.config.cloud.dialog_already_connected.ip_address"
+                )}:
+              </p>
+              <p>${details.remote_ip_address}</p>
+            </div>
+            <div class="instance-detail">
+              <p>
+                ${this.hass.localize(
+                  "ui.panel.config.cloud.dialog_already_connected.connected_at"
+                )}:
+              </p>
+              <p>
+                ${formatDateTime(
+                  new Date(details.connected_at),
+                  this.hass.locale,
+                  this.hass.config
+                )}
+              </p>
+            </div>
+          </div>
+          <ha-alert
+            alert-type="info"
+            .title=${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.info_backups.title"
+            )}
+          >
+            ${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.info_backups.description"
+            )}
+          </ha-alert>
+        </div>
+
+        <mwc-button @click=${this.closeDialog} slot="secondaryAction">
```

Comment:
```suggestion
        <ha-button @click=${this.closeDialog} slot="secondaryAction">
```

ha-button is mainly mwc-button, but when we want to migrate the buttons in future it's easier for us when we stick to `ha-button`